docs/apis/packaging-models.md

```diff
@@ -22,5 +22,46 @@ Specify `model` in an API, e.g.
 ```yaml
 - kind: api
   name: my-api
+  model_format: tensorflow
   model: s3://my-bucket/model.zip
 ```
+
+## ONNX
+
+Export your trained model to an ONNX model format. An example of a trained model being exported to ONNX is shown below.
+
+```Python
+...
+
+logreg_model = LogisticRegression(solver="lbfgs", multi_class="multinomial")
```

Comment:
Maybe show the `LogisticRegression` import?

Reply:
There are other things that need to be imported for this chunk of code to work like convert_sklearn, and FloatTensorType. If we want to have the imports I would rather just show the entire 20 lines of code.

Reply:
Maybe then use the fully qualified path, like `sklearn.LogisticRegression()`?